deployment/packagebuild/packages.d/vnet/openvnet-vnmgr/recipe.rb

```diff
@@ -18,11 +18,14 @@ class OpenvnetVnmgr < FPM::Cookery::Recipe
     /etc/openvnet/vnmgr.conf
   )
 
+  post_install 'post-install'
```

Comment:
Can you change the file name to `post-install.sh`? That will make it more clear that it's a shell script.